source/getting-started/configuration.markdown

```diff
@@ -9,19 +9,55 @@ sharing: true
 footer: true
 ---
 
-When launched for the first time, Home Assistant will write a default configuration file, called `configuration.yaml`, enabling the web interface and [device discovery](/components/discovery/). It can take up to a minute for your devices to be discovered and appear in the user interface.
+If you made it here, awesome! That means that you got Home Assistant up and running. It might have already found some of your networked devices. This is going to be fun!
 
-The `configuration.yaml` is written in [YAML](/docs/configuration/yaml/), stored in [`.homeassistant`](/docs/configuration/), and can be modified with a text editor.
+Home Assistant stores its configuration in a file called `configuration.yaml`. A default one is created when Home Assistant is started for the first time. Some of the things in the configuration file can be edited via the user interface, other parts require you to edit the configuration file directly.
 
-See the [components overview page](/components/) to find sample entries for your devices and services. For a sensor that is showing [random values](/components/sensor.random/), the entry would look like the sample below:
+There are two common approaches to edit your configuration: via Samba/Windows Networking and via the HASS Configurator. Both of these are [official add-ons for Hass.io](/addons/).
+
+## {% linkable_title Installing Hass.io add-ons %}
+
+Hass.io add-ons are installed from the add-on store embedded in the Hass.io panel:
+
+ - Open Home Assistant by navigating to [http://hassio.local:8123][local].
+ - Click on the menu icon in the top left and select Hass.io in the sidebar.
+ - The Hass.io panel opens, now open the add-on store by clicking the shopping bag.
+
+[local]: http://hassio.local:8123
+
+<p class='img'>
+<img src='/images/hassio/screenshots/main_panel_store_icon.png' />
+From the Hass.io main panel open the add-on store.
+</p>
+
+### {% linkable_title Editing config via HASS Configurator %}
+
+The first add-on we should install is the HASS Configurator. With the HASS Configurator you'll be able to edit your Home Assistant configuration from the web interface.
+
+Go to the add-on store (see previous step), click on Configurator and click on INSTALL. When installation is complete you will be able to click the "WEB UI" link to open the Web UI.
+
+Time for the first practice with the configurator. Add the following to your `configuration.yaml` file to add a link to the Configurator in the sidebar:
```

Comment:
location of configuration.yaml is missing: \\HASSIO\config\configuration.yaml

Reply:
They don't need the location since here they are using the web based editor to edit their configuration?

Reply:
maybe I'm blind but I see no web based editor (without installing another addon)

Reply:
nevermind, they can use the link as you wrote above :)

Reply:
2 issues about configurator:
-configurator is not running out of the box. Supervisor log file tells to set a password first (null -> "test1234").
-the gui is not beginner friendly:
  -configuration.yaml not preselected
  -no config check
  -many confusing buttons and links

I'm not sure if new users should use this

Reply:
It's the best we've got out of the box on a Hass.io installation.

Reply:
I've added your extra steps to the instructions. 

I am open to explore finding a better solution. Maybe we should enable the hass.io community addons by default and write instructions that work with Cloud9 add-on by @frenck ?